src/Files.App/Views/Layouts/DetailsLayoutPage.xaml.cs

```diff
@@ -607,12 +618,33 @@ clickedItem is Microsoft.UI.Xaml.Shapes.Rectangle
 			}
 		}
 
+		private async Task OpenItem(ListedItem item)
+		{
+			if (!Commands.OpenItem.IsExecutable)
```

Comment:
Can you please add a comment clarifying that this logic is for search results? This will help make sure we don't change this logic in the future 🙂